index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>HANGMAN</title>
+    <meta name="description" content="hangman">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Coiny&amp;subset=latin-ext" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class = 'container'>
+      <div class = 'header'>
```

Comment:
you could add empty lines between components of the same level of nesting to have better readability. 

Only exception would be the elements written in one line (look below)